### PR TITLE
Create a security overview public page

### PR DESCRIPTION
**Pull request by eth2devops**
## Summary
We need to create a security overview of public docs

## Details
The security section includes these pages: Security Overview, Roadmap, and Contacts

ticket: #96

### CODE REVIEW

**Comment by eth2devops** [comment]
@thomasheremans @OisinKyne this is the security public docs (overview, roadmap, and contacts) please have a look

File: docs/sec/overview.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+1. [Obol Audits](#obol-audits)
+2. [Open Problems and Challenges](#open-problems-and-challenges)
+3. [Core Public Goods](#core-public-goods)
+4. [List of Security Audits](#list-of-security-audits)
+
+## Obol Audits
+Obol Network has completed the following security audits:
+- Ethereal Ventures Security Audit [Report](https://hackmd.io/sH7FDtgTQV-2jN2ODfiscw)
+
+## Open Problems and Challenges
```
Comment (thomasheremans) [inline docs/sec/overview.md:21]:
```suggestion
## Open Challenges
```

File: docs/sec/overview.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+# Overview
+This page serves as an overview of the Obol Network from a security auditor’s point of view. It lists all of the projects that are intended to fall under the scope of the Obol Network project, as well as past audit reports, notable security bugs, and open security/privacy challenges in the Obol Network. You can think of this page as “a security auditor’s guide to Obol.”
+
+This page is updated quarterly. The last update was on 2023-03-21.
+
+## Table of Contents
+1. [Obol Audits](#obol-audits)
+2. [Open Problems and Challenges](#open-problems-and-challenges)
```
Comment (thomasheremans) [inline docs/sec/overview.md:13]:
```suggestion
2. [Open Challenges](#open-problems-and-challenges)
```
I find the Problem wording a bit too strong

**Review by thomasheremans — APPROVED** [review]

File: docs/sec/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+description: Security Roadmap
+sidebar_position: 2
+---
+
+# Roadmap
+
+## Past Audits
+Obol Labs has completed the following security audits:
+1. Ethereal Ventures - Jan 25, 2023 - [Audit report](https://hackmd.io/sH7FDtgTQV-2jN2ODfiscw)
+
```
[the remark this answers is not in the document]
Comment (eth2devops) [reply docs/sec/roadmap.md:11]: @thomasheremans do you know the details to include it? or we can push this PR and then update again later when the audit is over and the report is sent back to us for the auditors.

**Pull request merged**